odl/solvers/util/steplen.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, function, tau=0.5, discount=0.01, alpha=1.0,
         ----------
         function : callable
             The cost function of the optimization problem to be solved.
-            If function is not a `Functional`, the argument `dir_derivative`
+            If function is not a `Functional`, the argument ``dir_derivative``
```

Comment:
This looks like a hanging sentence here? Obviously from before.

Reply:
I'll fix it.

Reply:
I'll trust your judgement on this, but two if statements following each other is typically read as independent. Even doing elif would make it read as related without changing the results

Reply:
I'm gonna check, but I'm not entirely positive that the two are mutually exclusive.

Reply:
Hence the use of judgement :)

Reply:
Well, since they return immediately anyway, it doesn't matter. I'll use elif then.